platforms/ide/tooling-api/src/main/java/org/gradle/tooling/internal/protocol/InternalProblemAggregationDetails.java

```diff
@@ -18,6 +18,10 @@
 
 import org.gradle.api.NonNullApi;
 
+/**
```

Comment:
For future reference, I think we should add a comment here, that we kept the interfaces for maintaining cross-version compatibility with TAPI version X where these classes are still being used.

(true for all modified classes in the protocol package)